webpack.config.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
   module: {
     loaders: [
       {
-        test: /\.js(x|)?$/,
+        test: /\.js(x|)$/,
```

Comment:
Yes! I was tempted to change it to that – actually I didn't want to modify this file at all to keep the diff smaller; this change is a mistake. But I'll change it to that so the syntax matches what's in `karma.conf.js`.

The `loader` here is bugging me too; in the Karma config it's `babel?stage=1` and here it's `babel-loader?stage=0`. The former syntax is what I'm seeing in recommended in documentation everywhere, do you know why this is different?